content/english/events/20220401.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Nothing New Under The Sky"
+date: 2022-04-01T00:00:41+07:00
+slug: "nothing-new-under-the-sky"
+description: "In this millenial era and new economic many people are feeling that we need to adapt with the workplace. Is it true? Why the Bible says \"Nothing New Under the Sky\"?"
+keywords:
+        [
+                "Fellowship",
+                "Weekly",
+                "Youth",
+                "GRII BSD",
+                "Pemuda",
+                "Christian",
+                "Faith",
+        ]
+tags: ["Fellowship"]
+math: false
+toc: true
+thumbnail: "/images/events/20220401.jpeg"
+draft: false
+---
+
+![image](/images/events/20220401.jpeg)
+
+In this millenial era and new economic many people are feeling that we need to adapt with the workplace. Is it true? Why the Bible says "Nothing New Under the Sky"?
+Get the answer in:
```

Comment:
ini mau dipisah baris atau mau as it is? kalo mau pisah baris kasih \ di line sebelomnya